extensions/readme.md

```diff
@@ -0,0 +1,51 @@
+# Jetpack Block Editor Extensions
+
+This directory lists extensions for the Block Editor, also known as Gutenberg, [that was introduced in WordPress 5.0](https://wordpress.org/news/2018/12/bebo/).
+
+## Extension Type
+
+We define different types of block editor extensions:
+
+- Blocks are available in the editor itself, and live in the `blocks` directory.
+- Plugins are available in the Jetpack sidebar that appears on the right side of the block editor. Those live in the `plugins` directory.
+
+When adding a new extension, add a new directory for your extension the matching directory.
+
+## Extension Structure
+
+Your extension should follow this structure:
+
+```
+.
+└── blockname/
+	├── blockname.php ← PHP file where the block and its assets are registered.
+	└── client/
+		├── editor.js   ← script loaded only in the editor
+		├── editor.scss ← styles loaded only in the editor
+		├── view.js     ← script loaded in the editor and theme
+		└── view.scss   ← styles loaded in the editor and theme
```

Comment:
Since this is still anticipated (and not actually happening), I'd leave the details out for now and add it once we actually move the block frontend assets here. What do you think?

Reply:
That works for me. I'll update the readme to remove this for now.

Reply:
Done in 77cecb187b6bb8295e9d63909173545ca983de6b

Reply:
Thanks. To elaborate more; I wanted to avoid any confusion until we have build steps and the rest in place. :-)